src/components/Composer/index.js

```diff
@@ -246,6 +250,16 @@ export class Composer extends Component {
 						onClick: this.handleClick,
 					}
 				)}
+
+				oncompositionstart={(e)=>{
```

Comment:
```suggestion
				onCompositionStart={(e)=>{
```
Use camelCase to write event names: https://reactjs.org/docs/events.html#composition-events